src/nexus/GUI.py

```diff
@@ -173,6 +173,9 @@ def refresh(self):
                 item = QTableWidgetItem()
                 item.setData(Qt.ItemDataRole.DisplayRole, str(word.average_speed)[2:-3])
                 self.chentry_table.setItem(i, 3, item)
+                item = QTableWidgetItem()
+                item.setData(Qt.ItemDataRole.DisplayRole, str(len(word.word) * word.frequency))
```

Comment:
The `str` call here breaks the sort for the score column (it should be a numeric sort and not an alphabetic sort)